contracts/utils/Panic.sol

```diff
@@ -25,8 +25,8 @@ library Panic {
     uint256 internal constant GENERIC = 0x00;
     /// @dev used by the assert() builtin
     uint256 internal constant ASSERT = 0x01;
-    /// @dev arithmetic underflow or overflow
```

Comment:
This is correct, it's just that the source defines it like that:

https://github.com/ethereum/solidity/blob/v0.8.24/libsolutil/ErrorCodes.h#L29

I'm fine with only having `overflow`, though. Just mentioning since we may want to send a PR to update the source, he.

Reply:
Yes, hardhat chose to remove underflow since it is related to floating point not integers, solidity may follow the same path eventually

Reply:
Right, I think we should keep the comment you wrote. I readded it and I'm solving, just wanted to point out this as someone may come in the future with a PR updating it according to the source.